diff -Nru gnome-iconedit-1.0.0/Makefile.am gnome-iconedit-1.0.0.new/Makefile.am
--- gnome-iconedit-1.0.0/Makefile.am	Sat Jan 22 03:26:52 2000
+++ gnome-iconedit-1.0.0.new/Makefile.am	Sat Jan 29 14:52:19 2000
@@ -6,6 +6,6 @@
 docdir = $(datadir)/gnome/help/gnome-iconedit/C
 doc_DATA = topic.dat
 
-desktopdir = $(datadir)/gnome/apps/Graphics
+desktopdir = $(datadir)/applnk/Graphics
 desktop_DATA =	gnome-iconedit.desktop
 
