--- gnome-iconedit-1.2.0/gnome-iconedit.desktop.orig	Thu Mar  9 14:22:25 2000
+++ gnome-iconedit-1.2.0/gnome-iconedit.desktop	Wed Dec 17 20:54:09 2003
@@ -1,17 +1,21 @@
 [Desktop Entry]
 Name=GNOME Icon Editor 
 Name[de]=GNOME-Iconeditor
-Name[uk]=GNOME Редактор п╕ктограм
-Name[ko]=╠вЁП ╬фюлдэ фМаЩ╠Б 
-Name[ru]=Редактор иконок
-Name[tr]=Gnome ikona editЖrЭ
+Name[ko]=Й╥╦К├┬ Л∙└Л²╢Л╫≤ М▌╦Л╖▒Й╦╟ 
+Name[pl]=Edytor ikon GNOME
+Name[ru]=п═п╣п╢п╟п╨я┌п╬я─ п╦п╨п╬п╫п╬п╨
+Name[uk]=GNOME п═п╣п╢п╟п╨я┌п╬я─ п©я√п╨я┌п╬пЁя─п╟п╪
+Name[tr]=Gnome ikona editц╤rц╪
 Comment=Edits icons
 Comment[de]=Zum Bearbeiten von Icons
-Comment[uk]=Редактор п╕ктограм
-Comment[ko]=╬фюлдэю╩ фМаЩгу╢о╢ы
-Comment[ru]=Редактор иконок для GNOME
-Comment[tr]=IkonalarЩ deПiЧtirir
+Comment[ko]=Л∙└Л²╢Л╫≤Л²└ М▌╦Л╖▒М∙╘К▀┬К▀╓
+Comment[pl]=Edytor ikon dla GNOME
+Comment[ru]=п═п╣п╢п╟п╨я┌п╬я─ п╦п╨п╬п╫п╬п╨ п╢п╩я▐ GNOME
+Comment[uk]=п═п╣п╢п╟п╨я┌п╬я─ п©я√п╨я┌п╬пЁя─п╟п╪
+Comment[tr]=Ikonalarд╠ deд÷iе÷tirir
 Exec=gnome-iconedit
 Icon=gnome-iconedit/gnome-iconedit.png
-Terminal=0
+Terminal=false
 Type=Application
+Encoding=UTF-8
+# vi: encoding=utf-8
